deploy_local_testnet.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -i
+
+
+echo "0. Changing working directory as home folder"
+cd ~
+
+echo "1. Updating apt-get"
+sudo apt-get update -qq
```

Comment:
Does this script work only on deb-based distributions? Is this intentional? I think, you should at least to mention this somehow.